com.opentabletdriver.OpenTabletDriver.yaml

```diff
@@ -0,0 +1,69 @@
+app-id: com.opentabletdriver.OpenTabletDriver
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: start-otd.sh
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=all
+modules:
+  - name: libnotify
+    buildsystem: meson
+    config-opts:
+      - -Dman=false
+      - -Dgtk_doc=false
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libnotify.git
+        tag: 0.8.3
+
+
+  - name: libevdev
+    buildsystem: meson
+    config-opts:
+      - -Dtests=disabled
+      - -Ddocumentation=disabled
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: git
+        url: https://gitlab.freedesktop.org/libevdev/libevdev.git
+        tag: libevdev-1.13.1
+
+  - name: opentabletdriver
+    buildsystem: simple
+    build-options:
+      append-path: "/usr/lib/sdk/dotnet6/bin"
+      append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+      build-args:
+        - --share=network
+      env:
+        # PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+      append-pkg-config-path: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+    build-commands:
+      - ./eng/linux/package.sh -o bin -c Release -r linux-x64 -- --self-contained true --source nuget-sources
+      - install -Dm644 com.opentabletdriver.OpenTabletDriver.desktop /app/share/applications/com.opentabletdriver.OpenTabletDriver.desktop
+      - install -Dm644 256.png /app/share/icons/hicolor/256x256/apps/com.opentabletdriver.OpenTabletDriver.png
+      - install -Dm755 start-otd.sh /app/bin/start-otd.sh
+      - mv /run/build/opentabletdriver/bin/* /app/bin/
+    sources:
+      - sources.json
+      - type: git
+        url: https://github.com/OpenTabletDriver/OpenTabletDriver
+        tag: v0.6.4.0
+      - type: file
+        path: com.opentabletdriver.OpenTabletDriver.desktop
+      - type: file
+        path: 256.png
```

Comment:
so it doesn't even have an icon upstream?

Reply:
I have explained it